Tasks/XamariniOSV2/Tests/L0.ts

```diff
@@ -14,6 +14,24 @@ describe('XamariniOS L0 Suite', function () {
 
     });
 
+    it('run XamariniOSV2 with all default inputs', (done) => {
```

Comment:
This needs to be a `function` for `this.timeout(1000)` to do anything.

Reply:
Good point, I recollect now that timeouts don't apply otherwise. Will fix in my next PR.